models/unit/unit.go

```diff
@@ -193,6 +193,14 @@ func (u Unit) IsLessThan(unit Unit) bool {
 	return u.Idx < unit.Idx
 }
 
+// MaxPerm returns the max perms of this unit
+func (u Unit) MaxPerm() perm.AccessMode {
+	if u.Type == TypeExternalTracker || u.Type == TypeExternalWiki {
+		return perm.AccessModeRead
+	}
+	return perm.AccessModeAdmin
```

Comment:
umm, we should never *fall back* to admin access, this code will not age well.

Reply:
It's not fallback. Or we can have a map to store every unit's max permission.

Reply:
It's the max permission any user could have for that type of unit.

I agree and can see that this is clearly badly named and very confusing.

Reply:
@zeripath Oh, now I understand the purpose of this func, thanks for clarifying